src/_cert_renew.sh

```diff
@@ -0,0 +1,35 @@
+# Build functionality for certificate renewal / injection into NGINX
+
+function mod_cert-renew() {
+  title "PlexTrac SSL Certificate Renewal"
+  # Check if using LETS_ENCRYPT
+  LETS_ENCRYPT_EMAIL=${LETS_ENCRYPT_EMAIL:-}
+  USE_CUSTOM_CERT=${USE_CUSTOM_CERT:-false}
+  if [ "$LETS_ENCRYPT_EMAIL" != '' ] && [ "$USE_CUSTOM_CERT" == 'false' ]; then
+    # IF LETS_ENCRYPT = TRUE
+    # ASK TO REMOVE pem/key
+    info "Let's Encrypt certificate detected!"
+    info "Would you like to force renew your SSL certificate?"
+    if get_user_approval; then
+      #if yes, continue to delete pem/key and recreate NGINX
+      info "Deleting current certificates and loading new certificates"
+      compose_client rm -f plextracnginx
+      docker volume rm plextrac_letsencrypt
```

Comment:
In most cases we can simply renew without needing to nuke our local config. This is a bit heavy-handed; if you want to force renewal even before the 30d expiration window this would be better handled w/ invoking acme.sh (probably).

Reply:
Basically, if you force-recreate the NGINX container, if your certificates are still valid, it will simply remount them and go on its way. Either I need to:
1. Remove the volume entirely (most dramatic)
1. Remove the certificates from the volume so they can be regenerated by NGINX
1. Exec into the container, and run acme.sh with force renew

I wrote this at night time, so I'll rework to use Acme to force-renew the certs from inside the NGINX

Reply:
More specifically - do we have a use case for forcing lets encrypt renewal? If that's not a common use case then we should avoid making that the default behavior here.

Reply:
Removed ability to force renew